src/lib.rs

```diff
@@ -59,8 +59,11 @@ mod macros;
 
 mod auth;
 mod dsn;
+#[cfg_attr(feature = "cargo-clippy", allow(cast_lossless))]
 mod project_id;
+#[cfg_attr(feature = "cargo-clippy", allow(cast_lossless))]
```

Comment:
This is really annoying, but the following does not work (and there is an open issue for that somewhere):
```
cargo +nightly clippy --all-features -- -D clippy -A cast_lossless
```